CGI/load_stage.sql

```diff
@@ -0,0 +1,151 @@
+/**************************************************************************
+* Copyright 2016 Observational Health Data Sciences and Informatics (OHDSI)
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+* Authors: Varvara Savitskaya, Vlad Korsik, Alexander Davydov
+* Date: Fall-2022
+**************************************************************************/
+
+DO $_$
+BEGIN
+	PERFORM VOCABULARY_PACK.SetLatestUpdate(
+	pVocabularyName			=> 'CGI',
+	pVocabularyDate			=> current_date,
+	pVocabularyVersion		=> 'CGI'||TO_CHAR(CURRENT_DATE,'YYYYMMDD'),
+	pVocabularyDevSchema	=> 'dev_cgi',
+	pAppendVocabulary		=> FALSE
```

Comment:
you can omit this clause,  because one vocabulary

Reply:
done